README.md

```diff
@@ -1,3 +1,10 @@
+
```

Comment:
Initially, I cloned down your repo and had no idea that it contained a second app called `imagearrayapp`.  The entire app is nested inside the other (`imagearrayapp` is inside `app`).  Inside `26-simple-android-apps` you should have two directories: one for each app, and they should both be named appropriately (`app` is not very useful, right?).